rpcserver.go

```diff
@@ -2720,6 +2720,8 @@ func (r *rpcServer) SubscribePeerEvents(req *lnrpc.PeerEventSubscription,
 			if err := eventStream.Send(event); err != nil {
 				return err
 			}
+		case <-eventStream.Context().Done():
```

Comment:
nit: pre-existing, but if we touch this, can we add a newline before each `case` statement here and a few more places in this commit?

Reply:
sure, will add now 👍